polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -139,8 +139,21 @@ object Security {
   implicit val decoder: Decoder[Security] = deriveConfigDecoder
 }
 
+final case class HotkeyInfo(
```

Comment:
I'm a little confused about `HotkeyInfo` here. A user can't add any new functionality with this config - all they can do is add a new mapping from keybinding -> hotkey. So `description` doesn't really seem useful here, and `hide`/`vimOnly` are internal implementation details to support vim mode, so they are definitely not relevant here. 

Maybe there should just be a set of allowed hotkeys (`MoveUp`, `MoveDown`, `RunActive`....), and the config just needs to be a mapping from key to binding? A set of allowed keys would also help users determine which 

```
ui:
  custom_keybindings:
    RunActive: 
      - Ctrl-Enter
      - Cmd-Enter
```
(Note - I changed the order from Hotkey -> binding, and added the option of setting multiple bindings. I did this to emulate other hotkey bindings I'm familiar with, like IntelliJ, where you select the thing you want to do first and then choose how you want to do it and can set up multiple hotkeys if you like). 

Thoughts @zzeekk @jeremyrsmith ?